ta/pkcs11/src/processing_asymm.c

```diff
@@ -1193,6 +1193,7 @@ static enum pkcs11_rc unwrap_rsa_aes_key(struct active_processing *proc,
 	rsa_key_size = info.keySize / 8;
 	wrapped_key_size = data_sz - rsa_key_size;
 	target_key_size = wrapped_key_size - 8;
+	aes_key_size = rsa_key_size;
```

Comment:
Looking at your change, whether it's applied or not, I think we are missing a test after line 1206, to at least check that the decrypted temporary key has the expected size:
```patch
 	res = TEE_AsymmetricDecrypt(proc->tee_op_handle,
 				    NULL, 0,
 				    data, rsa_key_size,
 				    aes_key_value, &aes_key_size);
 	if (res) {
 		rc = tee2pkcs_error(res);
 		goto out;
 	}
+	if (aes_key_size != ctx->aes_key_bits / 8) {
+		DMSG("Unexpected temporary key size %zu, expecting %"PRIu32,
+		     aes_key_size, ctx->aes_key_bits / 8);
+		return PKCS11_CKR_WRAPPED_KEY_INVALID;
+	}